Makefile

```diff
@@ -19,7 +19,7 @@ HEX_DIR		?= $(OUTPUT_DIR)/hex
 LOG_DIR		?= $(OUTPUT_DIR)/logs
 
 # Path to the keil binaries
-KEIL_PATH	?= ~/Downloads/keil_8051/9.60/BIN
+KEIL_PATH	?= ~/.wine/drive_c/Keil_v5/C51/BIN
 
 WINE_BIN	?= /usr/local/bin/wine
```

Comment:
Would be fine if you removed this and we just expect the wine command to be in path.
I think this was to support the simplicity studio thing, but that can be solved with wineprefix if people insist on not using docker.

Reply:
I think you can remove the symlink from Dockerfile then?

Reply:
I think this is what @saidinesh5 was also referring too. Yes, I will look into that.